packages/kbn-proc-runner/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@kbn/proc-runner",
```

Comment:
I think we should create a `@kbn/tooling-utils` package or something. Small packages are nice, but there are a couple other things like the [ToolingLog](https://github.com/elastic/kibana/blob/master/src/dev/tooling_log/tooling_log.js) that I'd like to share and since they'll all need a build step and such we might as well bundle them together, no?

Reply:
@kjbekkelund how to do you feel about having a `tooling-utils` package rather than separate packages for proc-runner, tooling-log, and other things (I'm thinking maybe some utils for getopts, error handling, etc.)

Reply:
I like it

Reply:
If we go down this route, can we please scope this project to only creating two tools in tooling-utils? I don't want to go down another rabbit hole. Remember, the point is to get new-platform merged, not to create packages.

ToolingLog tests reference code outside of the ToolingLog directory, from the kibana/utils directory, so would you be okay if I only pull in what those tests require, and no more? If this wouldn't be acceptable, we should not do this effort at all; and instead have a proc-runner package to start getting new-platform out there. Then the other dev tools can be ported over into a tooling-utils as they are needed. Thoughts @spalger @kjbekkelund ?

Reply:
Alternatively, we can keep `@kbn/proc-runner` as is, referencing tooling_log from kibana `src/` as `@kbn/plugin-helpers` does today. That's a slight incremental improvement we need to create better integration testing around http server.

And then create another issue for having more utilities inside of @kbn/utils.

I'm all for moving quickly here.

Reply:
I don't think we should pull in anything but proc runner for now, but I just want to have a stable package name and thing `@kbn/tooling-utils` is a good start.

The stream utils that tooling log uses are another thing I regularly want to use in other places, and I plan to pull those out into something like `@kbn/utils`, but I think it should be separate from tooling utils since those are things that can and must ship with Kibana (whereas tooling utils won't and shouldn't).

Maybe it's not worth creating two packages and we should just have one utils package, but either way I'd prefer not to create a proc-runner package just to rename it later to a more generic name and break all references to the original package.

Reply:
Gotcha, sounds good. I just pulled tooling-log and proc-runner into something called `@kbn/utils` right now, but I can rename it. I feel like tooling-utils is a redundant name. Tools and utilities are the same things. What about `@kbn/dev-utils`, so it's not yet another "dev-tools"? So we can still have `@kbn/utils` available for something that does ship with kibana. @spalger

Reply:
Also maybe @kjbekkelund has opinion on the name there.